data/forms/case-notes-recording.jsx

```diff
@@ -1,7 +1,8 @@
 import CASE_NOTE_TYPES from 'data/caseNoteTypes';
 export default {
-  title: '',
+  title: 'Case Notes Recording',
   path: '/form/case-notes-recording/',
+  includeDetails: true,
```

Comment:
we can have this in the `FormCasesWrapper`, as it has been created for the purpose to wrap all the cases form.